release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -2504,3 +2504,40 @@ link:https://access.redhat.com/solutions/5780621[{product-title} 4.5.31 containe
+
+Issued: 2021-03-03
+
+{product-title} release 4.5.33 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2021:0428[RHSA-2021:0428] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2021:0429[RHSA-2021:0429] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/5836141[{product-title} 4.5.33 container image list]
+
+[id="ocp-4-5-33-features"]
+==== Features
+
+[id="ocp-4-5-33-insights-operator-enhancements"]
+===== Enhancements to Insights Operator
+
+With this update, the Insights Operator now collects information from the `MachineConfigPools` cluster. This information is useful for troubleshooting. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1887763[*BZ#1887763*].
+
+[id="ocp-4-5-33-bug-fixes"]
+==== Bug fixes
+
+* Previously, an incorrect OVN-kubernetes security rule blocked certain inbound connections. While rare, some attempts to connect to a pod would fail spuriously. This update fixes the iptables to block the intended connections, which results in no spurious failures. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1921283[*BZ#1921283*])
+
+* Previously, a watch cache in the Kubernetes API was initialized from the global revision (`etcd`) and could remain for an undefined period if no changes were made. This behavior sometimes led to a situation in which a client got a resource version (RV) from a server that had observed a newer RV, disconnected from it due to a network error, and reconnected to a server that was behind, resulting in `Timeout: Too large resource version` errors. With this release, the reflector is fixed so that it can recover from these errors, and Operators that use the `client-go` library for getting notifications from the server can recover and make progress upon receiving the errors. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1877346[*BZ#1877346*])
+
+* Previously, attempts to write to a nil writer may have led to `invalid memory address` or `nil pointer dereference` errors. Sharing the same instance of the writer might have also led to an `index out of range [43] with length 30 and recovered from err index > windowEnd` error. This update fixes a data race in the `SerializeObject` function of the kube-apiserver. link:https://bugzilla.redhat.com/show_bug.cgi?id=1879208[*BZ#1879208*]
+
+* Previously, a misplaced array index while pruning records from memory would cause excessive memory usage and fail to remove old reports from the archive. This update changes the array index key so that prune successfully removes records from memory without causing excessive memory usage. link:https://bugzilla.redhat.com/show_bug.cgi?id=1894243[*BZ#1894243*]
```

Comment:
```suggestion
* Previously, a misplaced array index while pruning records from memory would cause excessive memory usage and fail to remove old reports from the archive. This update changes the array index key so that prune successfully removes records from memory without causing excessive memory usage. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1894243[*BZ#1894243*])
```